dot/parachain/backing/statement_table.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2023 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package backing
+
+import parachaintypes "github.com/ChainSafe/gossamer/dot/parachain/types"
+
+type Table interface {
```

Comment:
We need to open an issue to implement this interface. Would you like to create an issue to implement statement table?

Reply:
we already have issue for this https://github.com/ChainSafe/gossamer/issues/3471